.env.local.example

```
@@ -0,0 +1,49 @@
+
+# ASP.NET Core Configuration
+ASPNETCORE_ENVIRONMENT=Development
+PATHFINDER_PORT=8080
+RPC_PORT=8081
+
+POSTGRES_CONNECTION_STRING="Host=localhost;Port=5432;Database=postgres;Username=postgres;Password=postgres"
+POSTGRES_READONLY_CONNECTION_STRING="Host=localhost;Port=5432;Database=postgres;Username=postgres;Password=postgres"
+
+# PostgreSQL credentials (for run-index.sh)
+POSTGRES_USER=postgres
+POSTGRES_PASSWORD=postgres
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The POSTGRES_PASSWORD key should go before the POSTGRES_USER key

(UnorderedKey)